### PR TITLE
📌 Fix flower Docker image, pin version

### PR DESCRIPTION
**Pull request by sanggusti**
Lowering flower version to solve issue #294

### CODE REVIEW

**Comment by FarDust** [comment]
This works for me, as temporal solution

**Comment by JakNowy** [comment]
Works for me as well, please merge this.

**Comment by sanggusti** [comment]
Hi @tiangolo can you check my pr?

File: {{cookiecutter.project_slug}}/docker-compose.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@ services:
     # You also have to change the flower command
   
   flower:
-    image: mher/flower
+    image: mher/flower:0.9.4
```
Comment (fisher60) [inline {{cookiecutter.project_slug}}/docker-compose.yml:115]: As of at least today, the more recent tag for flower `0.9.7` works for this as well. Maybe worth bumping the version number. 
```suggestion
    image: mher/flower:0.9.7
```

**Comment by tiangolo** [comment]
Great, thank you @sanggusti! 🍰 

And thanks for the comments @FarDust, @JakNowy, @fisher60 ☕

**Pull request merged**